core/src/main/scala-3/chisel3/SelectableCompat.scala

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+trait SelectableCompat extends scala.reflect.Selectable
```

Comment:
A type alias would be better:
```suggestion
type SelectableCompat = scala.reflect.Selectable
```

Reply:
Also maybe better to call it `ReflectSelectable` to make it a bit more obvious (since we also have the `Selectable` ~polyfill~ shim in Scala 2).